Client/Ecosia/UI/EmptyBookmarksView.swift

```diff
@@ -0,0 +1,269 @@
+                    self.importBookmarksButton.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+                    view.heightAnchor.constraint(equalToConstant: UX.ButtonAreaHeight)
+                ])
+            }
+            centerView.addSubview(buttonsContainer)
+            NSLayoutConstraint.activate([
+                buttonsContainer.centerXAnchor.constraint(equalTo: centerView.centerXAnchor),
+                buttonsContainer.bottomAnchor.constraint(equalTo: centerView.bottomAnchor),
+                buttonsContainer.topAnchor.constraint(equalTo: centerView.topAnchor)
+            ])
+        }
+        
+        containerStackView.addArrangedSubview(buttonsContainerCenterView)
+        
+        // setup buttons
+        learnMoreButton.addTarget(self, action: #selector(onLearnMoreTapped), for: .touchUpInside)
+        importBookmarksButton.addTarget(self, action: #selector(onImportTapped), for: .touchUpInside)
+
+        applyTheme()
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(applyTheme), name: .DisplayThemeChanged, object: nil)
+    }
+    
+    private func addSection(imageNamed: String, text: String, listItems: [String]? = nil) {
+        // first section (tap the bookmark icon when you find a page you want to share)
+        let sectionStackView = UIStackView()
+        sectionStackView.axis = .horizontal
+        sectionStackView.alignment = .top
+        
+        if traitCollection.userInterfaceIdiom == .pad {
+            sectionStackView.addArrangedSubview(createSpacerView(width: UX.SectionSpacerWidth))
+        }
+        
+        let sectionIcon = UIImageView()
+        sectionIcon.tintColor = .theme.ecosia.secondaryText
+        sectionIcon.contentMode = .scaleAspectFit
+        sectionIcon.image = UIImage.templateImageNamed(imageNamed)
+        sectionIcon.setContentCompressionResistancePriority(.required, for: .horizontal)
+        sectionIcon.setContentHuggingPriority(.required, for: .horizontal)
+        sectionIcon.translatesAutoresizingMaskIntoConstraints = false
+        sectionIcon.widthAnchor.constraint(equalToConstant: UX.SectionIconWidth).priority(.required).isActive = true
+        
+        sectionStackView.addArrangedSubview(sectionIcon)
+        
+        let sectionOneIconLabelSpacer = UIView.build {
+            $0.widthAnchor.constraint(equalToConstant: UX.SectionIconLabelSpacerWidth).isActive = true
+        }
+        sectionStackView.addArrangedSubview(sectionOneIconLabelSpacer)
+        
+        let sectionLabelsStackView = UIStackView()
+        sectionLabelsStackView.axis = .vertical
+        sectionStackView.addArrangedSubview(sectionLabelsStackView)
+
+        let sectionLabel = UILabel()
+        sectionLabel.font = UX.SectionLabelFont
+        sectionLabel.numberOfLines = 0
+        sectionLabel.textColor = .theme.ecosia.secondaryText
```

Comment:
..also this is overridden in `applyTheme()`, isn't it? Thus I'd generally advocate to remove setting colors outside of `applyTheme()` to have a single point of truth 🧹

Reply:
Looks for this to work we actually another fix, as currently the sub-stackview are not looped over, I've fixed this! Thank you.